CMakeLists.txt

```diff
@@ -103,6 +103,9 @@ set(EXECUTABLE_OUTPUT_PATH ${PROJECT_BINARY_DIR})
 
 option(ENABLE_SDL "Build the SDL port" OFF)
 option(ENABLE_WX "Build the wxWidgets port" ON)
+if(ENABLE_WX)
+    option(ENABLE_XSS "Check XSS support in the wxWidgets port" OFF)
```

Comment:
I don't think this is necessary, we can just automatically use it when available.